client/test/diagnostics/annotation_1.ion

```diff
@@ -0,0 +1 @@
+"annotation"::value
```

Comment:
why is this invalid?

Reply:
I think annotation must be a symbol, right?

Reply:
Yes. Sorry not enough coffee :)